storage/myisam/myisamchk.c

```diff
@@ -1365,7 +1365,7 @@ static void descript(MI_CHECK *param, register MI_INFO *info, char * name)
 	 key < share->state.header.uniques; key++, uniqueinfo++)
     {
       my_bool new_row=0;
-      char null_bit[8],null_pos[8];
+      char null_bit[8],null_pos[11];
```

Comment:
This looks like a potential CV and should be reported to Oracle. (https://bugs.mysql.com/)

Reply:
This is a a separate admin utility, thus I guess not a CV